servers/src/main/java/tachyon/master/lineage/meta/LineageStore.java

```diff
@@ -195,10 +195,13 @@ public synchronized Lineage getLineage(long lineageId) {
    * Reports an output file as lost.
    *
    * @param fileId the file id
-   * @return the lineage containing the output file
+   * @return the lineage containing the output file, null if no lineage outputs the given file
    */
   public synchronized Lineage reportLostFile(long fileId) {
     Lineage lineage = mOutputFileIndex.get(fileId);
+    if (lineage == null) {
+      return null;
+    }
```

Comment:
It's possible that the lost file may not associate with any lineage. I'll change the logic to let the caller check it before calling this function.